instrument/D17_Definition.xml

```diff
@@ -73,6 +79,16 @@
     <id val="100000"/>
     <id val="100001"/>
   </idlist>
+  <!--2 Slits-->
+  <type name="Slit"/>
```

Comment:
I don't think this line is needed, There is no component of type Slit.

Reply:
This is true. Lines 87 and 91 need Slit removed. This applies as well for the Figaro IDF, where I used the same function.